lib/Plans/BusinessPlan.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring\Plans;
+
+use WHMCS\Module\Server\Plesk360Monitoring\Plan;
+
+class BusinessPlan implements Plan
```

Comment:
I would declare all classes as `final` to prevent random `extends`. If something should be inherited a `abstract` class should be introduced for that.

Reply:
plus one, also wanted to mention this

Reply:
Fixed